python/src/addons/Index.pyx

```diff
@@ -92,3 +95,21 @@
             cpp_key_values.get().push_back(cpp_kv)
 
         self.inst.get().MSet(cpp_key_values)
+
+    def MSet(self, *args):
+        return call_deprecated_method("MSet", "mset", self.mset, *args)
```

Comment:
```suggestion
        return call_deprecated_method("MSet", "m_set", self.mset, *args)
```
?

Without understanding much :)

Reply:
`mset` is [redis](https://redis.io/docs/latest/commands/mset/). It is basically a bulk endpoint to set multiple key/values with one command. A _bulk_ endpoint.

What about renaming it to `bulk_set`?

Reply:
`bulk_set` is much better IMO.